spark/dl/src/main/scala/com/intel/analytics/bigdl/models/autoencoder/Autoencoder.scala

```diff
@@ -35,12 +35,12 @@ object Autoencoder {
     model
   }
 
-  def graph(classNum: Int): Module[Float] = {
+  def graph(classNum: Int, isDynamic: Boolean = false): Module[Float] = {
```

Comment:
Why do we need isDynamic for the models/examples? It should always be static graph, no?

Reply:
This is for test...

Reply:
But why is it included in user level examples?

Reply:
Because the test code depend on the user level example code... Should I copy the model definition out?

Reply:
Just copy the code out

Reply:
it's done